test/e2e/vsphere_test.go

```diff
@@ -399,30 +453,32 @@ func TestVSphereKubernetes123UbuntuCuratedPackagesAdotUpdateFlow(t *testing.T) {
 	runCuratedPackagesAdotInstallUpdateFlow(test)
 }
 
-func TestVSphereKubernetes122UbuntuCuratedPackagesClusterAutoscalerSimpleFlow(t *testing.T) {
+func TestVSphereKubernetes125UbuntuCuratedPackagesClusterAutoscalerSimpleFlow(t *testing.T) {
```

Comment:
Watch out for failed tests if there are any kubernetes version dependencies for 1.25 that could affect curated packages